build/bin/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+# This is documented here:
+# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
+
+if ! whoami &>/dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "${USER_NAME:-devworkspace-api}:x:$(id -u):$(id -g):${USER_NAME:-devworkspace-api} user:${HOME}:/sbin/nologin" >> /etc/passwd
```

Comment:
The default user name was `che-workspace-crd-operator` when it was initially generated (the file I forgot to commit).

`devworkspace-api` is the name of the package of the new Workspace Spec API.

Reply:
I assume it might be put here the same user name as the one used in the Dockerfile

Reply:
changed `devworkspace-api` to `che-workspace-crd-operator`